src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/EventSequence_Tests.cs

```diff
@@ -24,6 +24,7 @@ public void TestClick()
 
 		[Test]
 		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)] // Wasm is flaky #9080
```

Comment:
Could be related to https://github.com/unoplatform/uno/issues/13844? If so, might be worth noting that in the issue and/or the code comment

Reply:
Added the note to the linked issue